src/server/controllers/upload-rec-controller.ts

```diff
@@ -16,6 +16,9 @@ export function mapRecGameMetadata(data: RecordedGameMetadata) {
   mappedData.playerData = mappedData.playerData.filter(
     (_player, idx) => idx !== 0
   );
+  delete mappedData.commands;
+  delete mappedData.commandParserWarnings;
+  mappedData.playerData.forEach((playerData) => delete playerData.techTimes);
```

Comment:
Can you elaborate on what this is doing?

Reply:
At the moment these are fields that are part of the parser output but probably aren't useful for the recs page, though if anyone (admittedly if I'm being realistic it's probably only ever going to be me!) ends up using the parser code for other purposes then returning this stuff can be of interest. So far I've only really had reason to want to do that twice, to look at exactly what orders caused unexpected stuff to happen.

Maybe it's better to just never add this to the returned object or something? I'm not too sure.

Reply:
To your last point, the more elegant approach would be to use a Mongo pipeline aggregation and `project` the fields that are used by the front end services. I'm ok leaving what you have since I don't want to blow up the rec flow right now...but could be a future PR if you wanted to clean this up a bit.